Sources/Filters/General/LineFilter/example/index.js

```diff
@@ -0,0 +1,33 @@
+import vtkFullScreenRenderWindow from 'vtk.js/Sources/Rendering/Misc/FullScreenRenderWindow';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkLineFilter from 'vtk.js/Sources/Filters/General/LineFilter';
+import vtkMapper from 'vtk.js/Sources/Rendering/Core/Mapper';
+
+import vtkStar from './star';
+
+// ----------------------------------------------------------------------------
+// Standard rendering code setup
+// ----------------------------------------------------------------------------
+
+const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({
+  background: [0, 0, 0],
+});
+const renderer = fullScreenRenderer.getRenderer();
+const renderWindow = fullScreenRenderer.getRenderWindow();
+
+// ----------------------------------------------------------------------------
+// Example code
+// ----------------------------------------------------------------------------
+
+const lineSource = vtkStar.newInstance();
```

Comment:
Seems that you create a start but the variable's name is line.

Reply:
this example was deleted, star modified to go from drawing lines to drawing polys